manifest.xml

```diff
@@ -9,5 +9,4 @@
   <logo>http://</logo>
   <depend package="base/cmake" />
   <depend package="base/types" />
-  <depend package="drivers/hidapi" />
```

Comment:
Hmmm...for the matters of my understanding...
What is the difference of declare it here (on manifest) and declare it [here](https://github.com/Brazilian-Institute-of-Robotics/bir.flat_fish.br-package_set/pull/27/files#diff-b6b59d48fa2fdb96a88aaebe013619cdR2)?

> Then you should add the osdep dependency as well, right now you're only removing it.

It is done by adding `<rosdep name="libhidapi-dev" />` on the manifest file (since there is no more a package called `drivers/hidapi`